CourseStream/CourseStream.Data/Migrations/20231121172427_EnrolledStudentsTableAdded.Designer.cs

```diff
@@ -0,0 +1,78 @@
+﻿// <auto-generated />
+using CourseStream.Data;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Migrations;
+using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Metadata;
+
+#nullable disable
+
+namespace CourseStream.Data.Migrations
+{
+    [DbContext(typeof(CourseStreamDBContext))]
+    [Migration("20231121172427_EnrolledStudentsTableAdded")]
+    partial class EnrolledStudentsTableAdded
+    {
+        /// <inheritdoc />
+        protected override void BuildTargetModel(ModelBuilder modelBuilder)
+        {
+#pragma warning disable 612, 618
+            modelBuilder
+                .HasAnnotation("ProductVersion", "7.0.13")
+                .HasAnnotation("Relational:MaxIdentifierLength", 63);
+
+            NpgsqlModelBuilderExtensions.UseIdentityByDefaultColumns(modelBuilder);
+
+            modelBuilder.Entity("CourseStream.Data.CourseStream", b =>
+                {
+                    b.Property<int>("Id")
+                        .ValueGeneratedOnAdd()
+                        .HasColumnType("integer");
+
+                    NpgsqlPropertyBuilderExtensions.UseIdentityByDefaultColumn(b.Property<int>("Id"));
+
+                    b.Property<string>("CourseName")
+                        .IsRequired()
+                        .HasColumnType("text");
```

Comment:
Are there any validation requirements here? Example `CourseName` length? Or does this check perform somewhere higher?

Reply:
Can't answear your question right now. Should discuss on the meeting

Reply:
No validation requirements for Course Name.